string version_info
---
uint8 config_status